config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -885,11 +1029,36 @@ fields:
               region: content
               weight: 30
 
+    -   name: researchstudy_collaborator
+        content_type: research_study
+        label: Collaborator
+        type: chado_contact_by_role_type_default
+        description: The researcher(s) that collaborates on the research study — this field should be used only for researchers who do not fall under any other categories.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                linker_table: project_contact
+                linker_fkey_column: contact_id
+        settings:
+            termIdSpace: CO_010
+            termAccession: '0000348'
```

Comment:
While this term is correct there is a better one. Let's use [this term](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C84336) instead:

Collaborator
NCIT:C84336
Any of two or more parties working jointly towards a common goal.